scripts/patch-release/patch-release_test.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestModifyVersionFile(t *testing.T) {
```

Comment:
Can you convert that to a ginkgo test? You basically need: 

* A `suite_test.go` like this: https://github.com/cri-o/cri-o/blob/main/internal/version/suite_test.go
* A test layout like there: https://github.com/cri-o/cri-o/blob/main/internal/version/version_test.go#L11

Reply:
added this change as requested